readthedocs/core/symlink.py

```diff
@@ -245,7 +246,14 @@ def symlink_single_version(self):
         if default_version is None:
             return
 
-        self._log("Symlinking single_version")
+        try:
+            versions_qs = ((self.project.versions.protected(only_active=False)
+                            .filter(built=True)) |
+                           self.project.versions.protected(only_active=True))
+            version = versions_qs.get(slug=default_version)
+            self._log("Symlinking single_version: {0}".format(version.slug))
+        except Version.DoesNotExist:
+            version = None
```

Comment:
Ah yes, looks like this wasn't ported to use the new private symlink logic, should be easy enough.